organize/models.py

```diff
@@ -119,9 +119,19 @@ def deploy(self):
 
         self.change_status_to(DEPLOYED)
 
-        if Event.objects.filter(city=self.city,
-                                country=self.get_country_display()).exists():
-            event = copy_event(self.previous_event, self.date)
+        event = None
+        if self.previous_event:
+            previous_event = self.previous_event
+        else:
+            previous_event = (
+                Event.objects
+                .filter(city=self.city, country=self.country)
+                .order_by("-date")
+                .first()
+            )
+
```

Comment:
Previous discussion for reference: https://github.com/DjangoGirls/djangogirls/pull/369#pullrequestreview-15906366